pkg/asset/installconfig/gcp/validation.go

```diff
@@ -66,8 +67,36 @@ func ValidateInstanceType(client API, ic *types.InstallConfig, fieldPath *field.
 			errMsg := fmt.Sprintf("instance type does not meet minimum resource requirements of %d MB Memory", req.minimumMemory)
 			allErrs = append(allErrs, field.Invalid(fieldPath.Child("type"), instanceType, errMsg))
 		}
+	} else if index := strings.Index(instanceType, "custom-"); index != -1 {
+
+		// Ensure the CPU and MEMORY are defined in custom instance type names
+		if customConfig := strings.Split(instanceType[index+len("custom-"):], "-"); len(customConfig) == 2 {
+
+			// Ensure the custom CPU is a valid integer
+			if customCpus, err := strconv.ParseInt(customConfig[0], 10, 0); err != nil {
+				errMsg := "instance type not found"
+				allErrs = append(allErrs, field.Invalid(fieldPath.Child("type"), instanceType, errMsg))
+			} else if customCpus < req.minimumVCpus {
+				errMsg := fmt.Sprintf("instance type does not meet minimum resource requirements of %d vCPUs", req.minimumVCpus)
+				allErrs = append(allErrs, field.Invalid(fieldPath.Child("type"), instanceType, errMsg))
+			}
+
+			// Ensure the custom Memory is a valid integer
+			if customMemory, err := strconv.ParseInt(customConfig[1], 10, 0); err != nil {
+				errMsg := "instance type not found"
```

Comment:
similar suggestion as for CPUs. With the current error message we would have the message "instance type not found" repeated twice for an invalid instance type like "custom-x-x"

Reply:
No longer in current code.